packages/rich-text/src/index.js

```diff
@@ -4,32 +4,28 @@
 import './store';
 
 export { applyFormat } from './apply-format';
-export { charAt } from './char-at';
 export { concat } from './concat';
 export { create } from './create';
 export { getActiveFormat } from './get-active-format';
 export { getActiveObject } from './get-active-object';
-export { getSelectionEnd } from './get-selection-end';
-export { getSelectionStart } from './get-selection-start';
 export { getTextContent } from './get-text-content';
 export { isCollapsed } from './is-collapsed';
-export { isEmpty, isEmptyLine } from './is-empty';
+export { isEmpty, isEmptyLine as __unstableIsEmptyLine } from './is-empty';
```

Comment:
Renaming the symbol at this point to use the `__unstable` prefix will work as far as the doc generator is concerned because it uses the export statements to filter out the symbols to be ignored. However, I would lean towards renaming the symbol declaration as well. It may be benefitial as an urge to settle this API every time we work on it. Thoughts?

Reply:
Yeah, maybe it's good to do that. But the reason it's marked unstable is that it was never meant to be exported. It's only used by the `RichText` component, which should eventually live in these packages. So it's meant as an internal helper function, temporarily exported because these's one component left that's not been moved to this package yet.

Reply:
> It may be benefitial as an urge to settle this API every time we work on it. 

So, regarding this, there's nothing to be settled. We need to move `RichText` so we can remove the export statements.